setup.py

```diff
@@ -603,6 +612,11 @@ def bundle_libzmq_extension(self):
                 else:
                     info("ok")
 
+                if pypy:
+                    # seem to need explicit libstdc++ on linux + pypy
+                    # not sure why
+                    libzmq.libraries.append("stdc++")
```

Comment:
Excuse me for disturbing you but in #1161 I intentionally disabled adding ``libstdc++`` to explicitly linked libraries under PyPy. Do you have any reasons to enable it again? And if you do please tell me, may be I'm missing something.

Reply:
Uhmmm no. Likely my bad while merging.